web/js/feedback.js

```diff
@@ -8,7 +16,7 @@ wv.feedback = wv.feedback || (function () {
       'mailto:@MAIL@?subject=Feedback for @LONG_NAME@ tool');
 
     $element.click(function (event) {
-      if (!wv.util.browser.small && window.feedback) {
+      if (!util.browser.small && window.feedback) {
```

Comment:
I think `util.browser` is a separate file

Reply:
It is a separate file, but the `util` module imports the `browser` object to keep it all on the same parent object;

https://github.com/nasa-gibs/worldview/blob/53ef2980fa575135c68a2d53922c94cb89abd172/web/js/util/util.js#L7-L9

https://github.com/nasa-gibs/worldview/blob/53ef2980fa575135c68a2d53922c94cb89abd172/web/js/util/util.js#L16-L18

This is consistent with the behavior before where these objects were all inheriting from and extending the same `wv.util` object. It also makes updating the references easier and less error prone, and importing all of the submodules of `util` less verbose.